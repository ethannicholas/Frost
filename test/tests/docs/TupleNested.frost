method main() {
    def ((this, is), a, (nested, tuple)) := (("This", "is"), "a", ("nested", "tuple"))
    Console.printLine(this)
    Console.printLine(is)
    Console.printLine(a)
    Console.printLine(nested)
    Console.printLine(tuple)
}
