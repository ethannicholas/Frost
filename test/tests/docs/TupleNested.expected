This
is
a
nested
tuple
